### PR TITLE
updates on the using-the-API page on gen3.org

### PR DESCRIPTION
**Pull request by xritter1**
https://ctds-planx.atlassian.net/browse/PXP-7193

### Improvements
- Analyze-data: updated deprecated commands in the section "https://gen3.org/resources/user/analyze-data/#2-getting-files-into-the-gen3-workspace". 
- Analyze-data: updated the demo notebook that can be used to analyze data on the canine data commons.  

- Using-the-API: updated and extended the intro section.
- Using-the-API: updated deprecated API commands.
- Using-the-API: added another example for an API request.

- config and gen3.org: replaced the link to the "Sandbox" demo commons with the one to gen3.datacommons.io.
- submit-data: adding a note about comma-separated lists in nodes.submitter_id
- gen3-client: update links to public SDK
- access-data: update links to public SDK

### CODE REVIEW

**Review by svburke — APPROVED** [review]

**Review by wangfan860 — APPROVED** [review]
👍LGTM!

File: content/resources/user/using-api.md [hunk 1]
```diff
@@ -18,11 +18,18 @@ menuname: userMenu
...
+
+Further API specifications of the Gen3 mirocroservices can be browsed in [the developer documentation](/resources/developer/microservice) and (REST) API requests documentation of a Gen3 microservice can be found on swagger (e.g. [Sheepdog](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/uc-cdis/sheepdog/master/openapi/swagger.yml#/), [Peregrine](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/uc-cdis/peregrine/master/openapis/swagger.yaml)).
```
Comment (paulineribeyre) [inline content/resources/user/using-api.md:29]:
```suggestion
Further API specifications of the Gen3 microservices can be browsed in [the developer documentation](/resources/developer/microservice) and the (REST) API Swagger documentation for each Gen3 microservice can be found in the microservice's documentation on GitHub (e.g. [Sheepdog](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/uc-cdis/sheepdog/master/openapi/swagger.yml#/), [Peregrine](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/uc-cdis/peregrine/master/openapis/swagger.yaml)).
```

File: content/resources/user/using-api.md [hunk 2]
```diff
@@ -71,11 +81,22 @@ dl = requests.get(durl, headers=headers)
...
+        tsv = tsv + line + "\r"
+        count += 1
+program_name,project_code = <program_name>, <project_code>
+u = requests.put('https://gen3.datacommons.io/api/v0/submission/{}/{}'.format(program_name,project_code), data=tsv, headers=headers)
+u.text # should display the API response
+
 ```
 
-If an an error such as "You don't have access... " occurs, then the API key is most likely out of date and a new access token will need to be made.
+If an an error such as "You don't have access... " occurs, then the API key is most likely out of date and a new access token will need to be made. Further errors could occur if the uploaded file is not correctly formatted for the Gen3 data model.
```
Comment (paulineribeyre) [inline content/resources/user/using-api.md:99]:
```suggestion
If an an error such as "You don't have access... " occurs, then either you do not have access, or the API key is out of date and a new access token will need to be made. Further errors could occur if the uploaded file is not correctly formatted for the Gen3 data model.
```

**Review by cgmeyer — APPROVED** [review]
LGTM 🍕 🍺

**Pull request merged**